api/app/controllers/api/v1/applicants_controller.rb

```diff
@@ -34,6 +39,17 @@ def update
                 render_error(applicant.errors)
             end
         end
+
+        # /applicants/delete or /applicants/:id/delete
+        def delete
+            params.require(:id)
+            applicant = Applicant.find(params[:id])
+            if session.destroy!
```

Comment:
Why destroy the session? Aren't we destroying the applicant?

Reply:
Good catch. This was a mistake. I'll fix that. Thanks.

Reply:
Fixed.